### ISSUE TITLE
RFE: Toggle to more easily display building levels

### ISSUE THREAD

**Issue by SimonLandmine** [MegaMek/megamek#2524]
This came out of a discussion on Discord, where a user said that one of the things they find most challenging is quickly checking building heights in city fights - having to zoom in close to read the hex info, or mouse-over a range of buildings, to work out which ones you'll be able to Jump over, can be quite time-consuming when other players are waiting on you to finish your move.

A suggestion was a display toggle (including keyboard shortcut) that would display the building height on each building hex during the move turn (in a similar manner to the target modifiers during the firing turn). It would probably not be something players would want to leave on all of the time (hence the usefulness of a keyboard shortcut), but would make route-planning through (and over) cities a lot easier, and could then be switched off again.

**Comment by SJuliez** [comment]
Please take a look at #137 
This is a rather flexible system for displaying hex texts. It is still sitting on my hard drive because I was not happy with it. It is easily too cluttery on a complicated map and I will need to add something to make the texts display only when this and that is present in a hex. Or something. And I was thinking about displaying building info only on one hex of a building, not all. But I could finish it up and add it to the map editor only so we could take a look at it. And information clutter may be good in the map editor.

**Comment by SimonLandmine** [comment]
Yeah, concerns about information clutter were why I was thinking about having as something that could be easily toggled on and off, so as to be able to quickly check multiple building heights and then switched off again to reduce clutter.

That flexible system would enable users to customise the permanent display as desired, though - and if it could be easily toggled on and off with a key combination, it would enable the configuration of the 'extra info overlay' as desired.
